rs/nns/governance/CHANGELOG.md

```diff
@@ -11,4 +11,80 @@ here were moved from the adjacent `unreleased_changelog.md` file.
 INSERT NEW RELEASES HERE
 
 
+# 2025-02-03: Proposal 135063
```

Comment:
All of this green stuff was released on Jan 27, see https://dashboard.internetcomputer.org/proposal/134988

Reply:
Not quite!  Some of htis was Jan 27, some was Feb 3.  You need to look at the difference in proposals.

Reply:
I think these are straightened out now